src/api/app/models/workflow/step.rb

```diff
@@ -1,6 +1,8 @@
 class Workflow::Step
   include ActiveModel::Model
 
+  SHORT_COMMIT_SHA_LENGTH = 6
```

Comment:
The value should be `7`. See my other comment on how to make this work with `7`.

Reply:
Good catch!

Reply:
This still needs to happen.